crates/web-sys/src/features/gen_AudioWorkletNodeOptions.rs

```diff
@@ -109,6 +109,8 @@ impl AudioWorkletNodeOptions {
     #[doc = "Change the `outputChannelCount` field of this object."]
     #[doc = ""]
     #[doc = "*This API requires the following crate features to be activated: `AudioWorkletNodeOptions`*"]
+    #[doc = ""]
+    #[doc = "While the iterable or array can produce any JsValue as far as the type system is concerned, practically it is expected to contain a <code>[u32]</code>."]
```

Comment:
I think in this context `[u32]` probably needs a `&` in front to show it's a slice or it may not work because it's trying to hyperlink u32?

Reply:
This is a hyperlink; it'll look like "practically it is expected to contain a <code>[u32](https://doc.rust-lang.org/std/primitive.u32.html)</code>."

Reply:
Ah ok, I was getting the syntax confused a bit trying to read it.